source/clear-linux/guides/maintenance/increase-virtual-disk-size.rst

```diff
@@ -39,88 +39,88 @@ Hyper-V* image version number 20450, you can follow these steps:
 	                            "size" : "8G",
 	                            "type" : "linux" } ],
 
-The second method is to simply boot up your :abbr:`VM (Virtual Machine)` and
-execute the :command:`lsblk` command. Here's a sample output:
+The second method is to boot up your :abbr:`VM (Virtual Machine)` and
+execute the :command:`lsblk` command as shown below:
 
-	.. code-block:: bash
+.. code-block:: bash
+
+	sudo lsblk
 
-		# lsblk
+An example output: 
 
-   .. code-block:: console
+.. code-block:: console
 
-		NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
-		sda      8:0   0    8.5G  0 disk
-		├─sdd1   8:1   0    512M  0 part
-		├─sdd2   8:2   0     32M  0 part [SWAP]
-		└─sdd3   8:3   0      8G  0 part /
+	NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
+	sda      8:0   0    8.5G  0 disk
+	├─sdd1   8:1   0    512M  0 part
+	├─sdd2   8:2   0     32M  0 part [SWAP]
+	└─sdd3   8:3   0      8G  0 part /
 
-As you can see, both methods show the prebuilt Hyper-V image is about 8.5GB.
+In this example, both methods show the prebuilt Hyper-V image is about 8.5 GB.
 
 Increase virtual disk size
 **************************
 
-Follow these steps to increase the size of a prebuilt image:
+To increase the virtual disk size for a prebuilt image, perform the steps below:
 
-#.	Shutdown your VM if it's running.
+#.	Shut down your VM if it is running.
 #.	Use an appropriate hypervisor tool to increase the virtual disk size of
 	your VM.
 #.	Power up the VM.
-#. 	Login to an account with root privileges.
+#. 	Log in to an account with root privileges.
 #.	Open a terminal emulator.
 #.	Add the |CL| `storage-utils` bundle to install the `parted` and
 	`resize2fs` tools.
 
 	.. code-block:: bash
 
-		# swupd bundle-add storage-utils
+		sudo swupd bundle-add storage-utils
 
-#.	Launch the :command:`parted` tool.
+#.	Launch the `parted` tool.
 
 	.. code-block:: bash
 
-		# parted
+		sudo parted
 
 #.	In the `parted` tool, perform these steps:
 
-	#.	Press :kbd:`p` to print the partitions table.
-	#.	If you get the warning message:
+	#.	Press :command:`p` to print the partitions table.
+	#.	If the warning message below is displayed, enter :command:`Fix`.
 
 		.. code-block:: console
 
 			Warning: Not all of the space available to /dev/sda appears to be
-			used, you can fix the GPT to use all of the space (an extra ... 
+			used, you can fix the GPT to use all of the space (an extra ...
 			blocks) or continue with the current setting?
 
 			Fix/Ignore?
 
-		Enter :command:`fix`.
```

Comment:
@MCamp859  --  Changes look good. The only question I have is if you intended to delete this line (Enter :command:`fix`). It might be an artifact of the diff, but I wanted to verify.

Reply:
Thanks for checking -- the fix command was moved from line 96 to line 88.